core/trino-parser/src/main/java/io/trino/sql/tree/TableExecute.java

```diff
@@ -14,7 +14,6 @@
 package io.trino.sql.tree;
 
 import com.google.common.collect.ImmutableList;
-import org.checkerframework.checker.units.qual.C;
```

Comment:
I wonder why it was not reported by UnsedImports rule of checkstyle :/ @findepi ?

Reply:
because the class uses `C` as generic parameter name. i guess this apparently confused the checkstyle.

Reply:
> because the class uses C as generic parameter name. 

(for some methods)